### ISSUE TITLE
julia-doc: error while installing latest nightly (1896)

### ISSUE THREAD

**Issue by tpapp** [staticfloat/julia-debian#1]
```
$ sudo apt-get install julia-doc
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages will be upgraded:
  julia-doc
1 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
Need to get 30.2 kB of archives.
After this operation, 8,587 kB disk space will be freed.
Get:1 http://ppa.launchpad.net/staticfloat/julianightlies/ubuntu/ utopic/main julia-doc all 0.4.0-1896~ubuntu14.10.1 [30.2 kB]
Fetched 30.2 kB in 0s (160 kB/s)
Reading changelogs... Done
(Reading database ... 444330 files and directories currently installed.)
Preparing to unpack .../julia-doc_0.4.0-1896~ubuntu14.10.1_all.deb ...
Unpacking julia-doc (0.4.0-1896~ubuntu14.10.1) over (0.4.0-1845~ubuntu14.10.1) ...
Processing triggers for doc-base (0.10.6) ...
Processing 1 changed doc-base file...
Error in `/usr/share/doc-base/julia-manual', line 8: all `Format' sections are invalid.
Note: `install-docs --verbose --check file_name' may give more details about the above error.
Registering documents with scrollkeeper...
Setting up julia-doc (0.4.0-1896~ubuntu14.10.1) ...
```

Using latest debian unstable. Output from command suggested above:

```
$ install-docs --verbose --check /usr/share/doc-base/julia-manual 
Warning in `/usr/share/doc-base/julia-manual', line 8: file `/usr/share/doc/julia-doc/html/index.html' does not exist.
Error in `/usr/share/doc-base/julia-manual', line 8: all `Format' sections are invalid.
/usr/share/doc-base/julia-manual: Fatal error found, the file won't be registered.

```

**Comment by staticfloat** [comment]
Thank you for the error report.  I'll look into it.

**Comment by tpapp** [comment]
Thanks. The error is still there in version `0.4.0-1907~ubuntu14.10.1`.

I think that the problem is that `/usr/share/doc/julia-doc/html` is empty (except for `_static`, which is also empty), while before version `0.4.0-1845` it had the documentation (`index.html`, etc).

**Comment by staticfloat** [comment]
I think I figured out why this is happening.  I've pushed a fix (c4b6e3d6d665ab33af76003f036dc7735ef73f98) we'll see if it works in the next day or two, when a new `julia-doc` gets built on Canonical's servers.  (And thank you for the note, I just now saw it, but that is indeed the problem.  A clean rule is getting run before it gets packaged up, so the documentation I was uploading was getting cleaned before the build was actually performed!)
